Description: remove unused Sphinx extension
 This package uses a deprecated and later removed Sphinx extension
 that led to a FTBFS with recent Sphinx versions.
 Since it wasn't used in the documentation anyway, it should be safe to remove.
Author: Sascha Steinbiss <satta@debian.org>
Last-Update: 2019-02-09
--- a/doc/source/conf.py
+++ b/doc/source/conf.py
@@ -27,7 +27,7 @@
 
 # Add any Sphinx extension module names here, as strings. They can be extensions
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
-extensions = ['sphinx.ext.autodoc', 'sphinx.ext.doctest', 'sphinx.ext.intersphinx', 'sphinx.ext.todo', 'sphinx.ext.coverage', 'sphinx.ext.pngmath', 'sphinx.ext.ifconfig', 'sphinx.ext.viewcode']
+extensions = ['sphinx.ext.autodoc', 'sphinx.ext.doctest', 'sphinx.ext.intersphinx', 'sphinx.ext.todo', 'sphinx.ext.coverage', 'sphinx.ext.ifconfig', 'sphinx.ext.viewcode']
 
 # Breathe setup, for integrating doxygen content
 extensions.append('breathe')
